src/app/Recordings/ActiveRecordingsTable.tsx

```diff
@@ -129,7 +129,10 @@ export const ActiveRecordingsTable: React.FunctionComponent<ActiveRecordingsTabl
     addSubscription(
       combineLatest([
         context.target.target(),
-        context.notificationChannel.messages(NotificationCategory.ActiveRecordingCreated),
+        merge(
```

Comment:
What does the `merge` here do?

Reply:
Without the `merge` the web client doesn't properly update the table state upon receiving `ActiveRecordingDeleted` or `SnapshotDeleted` notifications. I included it here as well for consistency reasons even though the `ActiveRecordingCreated` and `SnapshotCreated` notifications don't exhibit the same behavior, although that's likely because a refresh is triggered upon returning from the `/create` route and not because the table is properly updating its state based on the notifications alone.

I think the reason why `combineLatest` doesn't work on its own is that it won't emit an initial value until each of the Observables comprising it doesn't emit at least once. For example, that means the first time an `ActiveRecordingDeleted` notification is received, `combineLatest` won't emit until it also receives a `SnapshotDeleted` notification, which leads to weird state behavior in the displayed table. 

However, by merging snapshot and active recording notification Observables into one, it ensures that `combineLatest` will emit soon as either a snapshot or active recording-related notification is received.

Reply:
Ahh, that makes sense. Thanks.